docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Hasura
+linkTitle: Hasura
+description: Hasura
+aliases:
+menu:
+  latest:
+    identifier: graphql-hasura
+    parent: ecosystem-integrations
+    weight: 575
+isTocNested: true
+showAsideToc: true
+---
+
+You can use the [Hasura GraphQL Engine](https://hasura.io) with YugabyteDB to power your GraphQL applications with a distributed SQL database.
+
+For more information, see the following:
+
+- [Hasura GraphQL Engine documentation](https://docs.hasura.io)
+- [Scaling GraphQL Subscriptions to One Million Subscribers Linearly](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/1M_subscriptions.md)
+- [High Availability of GraphQL Subscriptions Under Infrastructure Failures](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/resilient_graphql_apps.md)
+- [Multi-Region GraphQL Applications](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/multi_region_apps.md)
```

Comment:
Let's remove `Multi-region GraphQL Applications`, it is still under development.

Reply:
Deleted